client/src/app/recipe-details/recipe-details.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, Inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+
+
+@Component({
+    selector: 'app-recipe-details',
+    templateUrl: `./recipe-details.component.html`,
+    styleUrls: ['./recipe-details.component.css']
+})
+
+
+
+export class RecipeDetailsComponent {
+
+    public recipeDetails: RecipeDetails;
+
+    
+// Need to change this so recipe details are fetched only after selecting specific recipe from recipe search results
+    constructor(http: HttpClient, @Inject('BASE_URL') baseUrl: string) {
+        this.recipeDetails = {} as RecipeDetails;
+        http.get<RecipeDetails>(baseUrl + 'api/recipedetails').subscribe(result => {
+            this.recipeDetails = result;
+        }, error => console.error(error));
+    }
+}
+
+
+
+
+interface RecipeDetails {
```

Comment:
Just `Recipe` is good, it more or less means the same thing but fits better. Also, move this to the `RecipeService` or within that same directory, `client/src/services/recipe`.